README.md

```diff
@@ -1,3 +1,5 @@
+# This repository is moved to [this](https://github.com/sciencemesh/nc-sciencemesh) link.
+
```

Comment:
This shouldn't be merged upstream, should only be on https://github.com/pondersource/nc-sciencemesh